config/initializers/money.rb

```diff
@@ -0,0 +1,116 @@
+# encoding : utf-8
+require 'money/bank/open_exchange_rates_bank'
+
+MoneyRails.configure do |config|
+
+  # To set the default currency
+  #
+  config.default_currency = :usd
+  oxr = Money::Bank::OpenExchangeRatesBank.new(Money::RatesStore::Memory.new)
+  oxr.app_id = 'your app id from https://openexchangerates.org/signup'
+  oxr.update_rates
+  config.default_bank = oxr
+
+
```

Comment:
Keep this consistent by using only one blank line.

Reply:
Completed, removed one blank line